pages/[username].js

```diff
@@ -45,6 +46,11 @@ export async function getServerSideProps(context) {
     logger.error(e, `cannot strip markdown for: ${username}`);
     profile.cleanBio = profile.bio;
   }
+  try {
+    profile.profileStats = await getStats(username);
```

Comment:
After 1 final testing, I notice performance take a little hit. Maybe we can check for `isStatsPublic` here, and if `false` then we should not make this query, this will help with performance